UnitTests/GitCommandsTests/GitModuleTest.cs

```diff
@@ -63,5 +63,24 @@ public void ParseGitBlame()
             Assert.AreEqual("e3268019c66da7534414e9562ececdee5d455b1b", result.Lines.Last().Commit.ObjectId);
             Assert.AreEqual("", result.Lines.Last().Text);
         }
+
+        [Test]
+        public void UnescapeOctalCodePoints_handles_octal_codes()
```

Comment:
:bulb: Candidate for `TestCase`, which would break failing cases down to the specific ones impacted by a change.

Reply:
Done. The comments look a bit weird.